jazz_logs/index.js

```diff
@@ -88,14 +88,27 @@ module.exports.handler = (event, context, cb) => {
 
 			querys.push(utils.setQuery("servicename", service));
 			querys.push(utils.setQuery("environment", env));
-			querys.push(utils.setQuery("log_level", logType));
 
 			//Query to filter Control messages
 			querys.push(utils.setQuery("!message", "START*"));
 			querys.push(utils.setQuery("!message", "END*"));
 			querys.push(utils.setQuery("!message", "REPORT*"));
 
-			logger.info("QueryObj: "+ JSON.stringify(querys));
+			if (logType) {
+				var log_type_config = [];
+				_.forEach(config.LOG_LEVELS, function(value, key) {
+					log_type_config.push(value.Type);
+				});
+				
+				if(_.includes(log_type_config, logType.toLowerCase())) {
```

Comment:
includes is part of Array.prototype. please consider using that

Reply:
We are using NodeJs v4.3. Array.prototype is not supported for v4.3. It is supported from NodeJs v6.x and above. 

Polyfill may be used but don't want to use that to just replace  _.include  method for now.